### ISSUE TITLE
Better handling of responsive embeds

### ISSUE THREAD

**Issue by wiegand** [d-i/baseline#12]
We are currently using the technique [described here](http://www.smashingmagazine.com/2014/02/27/making-embedded-content-work-in-responsive-design/) for responsive embeds.

Let's extract this as a `mixin` that accepts a ratio. We will use this to build a set of standard radtios to include, e.g. 16×9, 4×3:

```sass
@mixin embed($width, $height) {
  padding-bottom: ($height / $width) * 100%;
  position: relative;
  padding: 0;
  height: 0;
  overflow: hidden;

  .embed__item,
  iframe,
  object,
  embed {
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
  }
}
```

Then used as:

```sass
.embed--16by9 {
  @include embed(16, 9);
}
```

**Comment by bdielman** [comment]
I was going to include this in the 2.0 release, but I would like to get this included sooner.

**Comment by bdielman** [comment]
Fixed by #13.
